mapie/multi_label_classification.py

```diff
@@ -742,10 +632,11 @@ def predict(
             Lower ``delta`` produce larger (more conservative) prediction
             sets.
             By default ``None``.
+
         bound : Optional[Union[str, ``None``]]
             Method used to compute the Upper Confidence Bound of the
             average risk. Only necessary with the RCPS method.
-            By default "wsr"
+            By default "wsr".
```

Comment:
According to your previous change (in function signature):
```suggestion
            By default ``None``.
```

Reply:
Done